docs/endpoints/multi_user.rst

```diff
@@ -295,7 +295,7 @@ motivated administrator.  The initial user config template implements two
 user-specifiable variables, ``endpoint_setup`` and ``worker_init``.  Both of these
 default to the empty string if not specified by the user (i.e., ``...|default()``).
 
-.. code-block:: yaml
+.. code-block:: yaml+jinja
```

Comment:
Ah, cool.  I was not aware of this syntax.

Reply:
I only ended up finding it because my new example caused an error without it. But it definitely looks much nicer 👍